app/controllers/application_controller.rb

```diff
@@ -1,2 +1,6 @@
 class ApplicationController < ActionController::Base
```

Comment:
- [ ] Please make sure that you fix the error below and make sure that all your routes are rendered without any error. Due to this error, I couldn't test your forms so ensure that your forms work perfectly  :pray: 

![image](https://github.com/matee-safi/Blog-app/assets/92303891/3977bc60-8346-4ac2-befe-950ab153b2ab)

Reply:
@Owusu-Desmond I have made some changes, can you check if it worked?